### DISCUSSION TITLE
Hyperion.ng

### DISCUSSION THREAD

**Discussion by Alessjo99** [tteck/Proxmox#849] [Ideas]
Please create a container for Hyperion.ng

**Comment by tteck** [comment]
Does Hyperion need USB passthrough? YES

**Comment by Alessjo99** [comment]
Perfect, thanks!

**Reply by liveware-problems** [reply]
@Alessjo99 Just curious what your use case for Hyperion LXC is? Can you run multiple detectors & controllers through a remote server for TVs in different rooms? I always thought the detectors would need to be local (USB?) attached to the Hyperion box, but I was intrigued when I spotted the Hyperion LXC and wondered how it could be used - thanks!
